native/cocos/core/assets/ImageAsset.h

```diff
@@ -104,16 +111,23 @@ class ImageAsset final : public Asset {
     inline void setFormat(PixelFormat format) { _format = format; }
     inline void setData(uint8_t *data) { _data = data; }
     inline void setUrl(const ccstd::string &url) { _url = url; }
+    inline void setMipmapLevelDataSize(ccstd::vector<uint32_t> &mipmapLevelDataSize) { _mipmapLevelDataSize = ccstd::vector(mipmapLevelDataSize);}
```

Comment:
No need to add `ccstd::vector(...)`. Just
```
_mipmapLevelDataSize = mipmapLevelDataSize;
```

Reply:
Will only copy references be accidentally freed?

Reply:
`_mipmapLevelDataSize = mipmapLevelDataSize;` is the same as `_mipmapLevelDataSize = ccstd::vector(mipmapLevelDataSize);`

`_mipmapLevelDataSize = mipmapLevelDataSize;` will invoke the copy-assignment to copy all elements to `_mipmapLevelDataSize`.

Reply:
> 

sure，thanks